service/history/workflow_task_handler.go

```diff
@@ -215,6 +215,12 @@ func (handler *workflowTaskHandlerImpl) handleCommand(
 	command *commandpb.Command,
 	msgs *collection.IndexedTakeList[string, *protocolpb.Message],
 ) (*handleCommandResponse, error) {
+
+	handler.metricsHandler.Counter(metrics.CommandCounter.GetMetricName()).Record(
+		1,
+		metrics.NamespaceTag(handler.mutableState.GetExecutionInfo().NamespaceId),
```

Comment:
should use namespace name, not namespace_id

Reply:
Also, handler.metricsHandler should be tagged with namespace when the handler is created.